josh/2021-12-01/main_d1_p1.py

```diff
@@ -0,0 +1,23 @@
+def read_file(file_path: str) -> list:
+    with open(file_path, 'r') as f:
+        raw_data = f.readlines()
+        depth_list = [int(x.rstrip()) for x in raw_data]
+    return depth_list
+
+
+def count_increases(depths: list) -> str:
+    counter = 0
+    for x in range(len(depths) - 1):
```

Comment:
This was where my error was. Still don't really understand why I need to take the len - 1 in order for this to work